computation/src/main/java/com/powsybl/computation/ComputationException.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.computation;
+
+import com.powsybl.commons.PowsyblException;
+
+import javax.annotation.Nullable;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+
+/**
+ * This class is responsible for carry out the .err/.out logs when execution failed.
+ *
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class ComputationException extends PowsyblException {
```

Comment:
As discussed just now I think that we should also have the full zip of logs, if it is available :
otherwise we will not have enough information in the exception to really identify the issue, if it was caused by a "deep call".
For example, if we run a `itools security-analysis --task N` and one of the sub-command fails, we should still get a zip of all available sub-commands logs.
Also, we must adapt the `SecurityAnalysisTool` so that it writes down the logs if they are available, when `runWithLog` throws.